gateways/db/postgres/entries/activity/list_activity.go

```diff
@@ -0,0 +1,89 @@
+package activity
+
+import (
+	"context"
+
+	"github.com/Vivi3008/apiTestGolang/domain/entities/bills"
+	"github.com/Vivi3008/apiTestGolang/domain/usecases/activities"
+	repoBil "github.com/Vivi3008/apiTestGolang/gateways/db/postgres/entries/bills"
+)
+
+type DestinyAccount struct {
+	AccountDestinationId string
+	Name                 string
+}
+
+type DescriptionPayment struct {
+	Description string
+	Status      bills.Status
+}
+
+func (r Repository) ListActivity(ctx context.Context, accountId string) ([]activities.AccountActivity, error) {
+	blRepo := repoBil.NewRepository(r.DB)
+
+	listBills, err := blRepo.ListBills(ctx, accountId)
+
+	if err != nil {
+		return []activities.AccountActivity{}, err
+	}
+
+	var listActivities = make([]activities.AccountActivity, 0)
+
+	if len(listBills) != 0 {
+		var actBl activities.AccountActivity
+		for i := 0; i < len(listBills); i++ {
+			actBl.Type = activities.Bill
+			actBl.Amount = listBills[i].Value
+			actBl.CreatedAt = listBills[i].ScheduledDate
+			actBl.Details = DescriptionPayment{
+				Description: listBills[i].Description,
+				Status:      listBills[i].StatusBill,
+			}
+			listActivities = append(listActivities, actBl)
+		}
+	}
```

Comment:
Esse if não é necessário. Se o slice for fazio, o for não fará nada. E eu usaria um for range aqui tipo `for _, bill := range listBills {`, mas aí é um pouco subjetivo
```suggestion
		for i := 0; i < len(listBills); i++ {
			var actBl activities.AccountActivity
			
			actBl.Type = activities.Bill
			actBl.Amount = listBills[i].Value
			actBl.CreatedAt = listBills[i].ScheduledDate
			actBl.Details = DescriptionPayment{
				Description: listBills[i].Description,
				Status:      listBills[i].StatusBill,
			}
			listActivities = append(listActivities, actBl)
		}
```